container/go/cmd/loader/loader.go

```diff
@@ -31,11 +33,24 @@ import (
 var (
 	outDir     = flag.String("directory", "", "Where to save the image's files.")
 	imgTarball = flag.String("tarball", "", "The path to the tarball to load.")
+	isGzip     = flag.Bool("gzip", false, "The tarball is gzip compressed.")
```

Comment:
Recommend detecting this automatically using [IsGzipped](https://github.com/google/go-containerregistry/blob/093adf2740a9617fbb6e6fe81ec37b00d3eb1b43/pkg/v1/v1util/zip.go#L82). This way if we want to support more compressed formats in the future, we won't have to add a flag + attribute for each of them.

Reply:
Done. That method is now moved to an internal package so can't be used directly from there. So I have added a new method to this PR which is similar.